src/sentry/static/sentry/app/components/events/eventCauseEmpty.jsx

```diff
@@ -0,0 +1,192 @@
+    }
+    if (data && data.snoozed_ts) {
+      return snoozedDays(data.snoozed_ts) > 7;
+    }
+    return true;
+  }
+
+  handleClick(action) {
+    const {api, project, organization} = this.props;
+
+    const data = {
+      projectId: project.id,
+      organizationId: organization.id,
+      feature: 'suspect_commits',
+      status: action,
+    };
+    promptsUpdate(api, data).then(this.setState({shouldShow: false}));
+  }
+
+  render() {
+    const {shouldShow} = this.state;
+    if (!shouldShow) {
+      return null;
+    }
+
+    return (
+      <div className="box">
+        <StyledPanel dashedBorder>
+          <BoxHeader>
+            <Description>
+              <h3>{t('Suspect Commits')}</h3>
+              <p>{t('Identify which commit caused this issue')}</p>
+            </Description>
+            <ButtonList>
+              <Button
+                size="small"
+                priority="primary"
+                href="https://docs.sentry.io/workflow/releases/#create-release"
+              >
+                {t('Read the docs')}
+              </Button>
+
+              <div>
+                <Tooltip title={t('Remind me in a week')}>
+                  <SnoozeButton
+                    size="small"
+                    onClick={() => this.handleClick('snoozed')}
+                    data-test-id="snoozed"
+                  >
+                    {t('Later')}
+                  </SnoozeButton>
+                </Tooltip>
+                <DismissButton
+                  size="small"
+                  onClick={() => this.handleClick('dismissed')}
+                  data-test-id="dismissed"
+                >
+                  {t('Dismiss')}
+                </DismissButton>
+              </div>
+            </ButtonList>
+          </BoxHeader>
+          <Panel>
+            <CommitRow
+              key={DUMMY_COMMIT.id}
+              commit={DUMMY_COMMIT}
+              customAvatar={robotLogo}
+            />
+          </Panel>
+        </StyledPanel>
+      </div>
+    );
+  }
+}
+
+const StyledPanel = styled(Panel)`
+  padding: ${space(3)};
+  padding-bottom: 0;
+  background: none;
+`;
+
+const BoxHeader = styled('div')`
+  display: grid;
+  align-items: start;
+  grid-template-columns: 1fr max-content;
+`;
+
+const Description = styled('div')`
+  h3 {
+    font-size: 14px;
+    text-transform: uppercase;
+    margin-bottom: ${space(0.25)};
+    color: ${p => p.theme.gray2};
+  }
+
+  p {
+    font-size: 14px;
+    font-weight: bold;
+    color: ${p => p.theme.gray4};
+    margin-bottom: ${space(1.5)};
+  }
+`;
+
+const ButtonList = styled('div')`
+  display: inline-grid;
+  grid-auto-flow: column;
+  grid-gap: ${space(1)};
+  align-items: center;
+  float: right;
+`;
+
+const SnoozeButton = styled(Button)`
+  border-right: 0;
+  border-top-right-radius: 0;
+  border-bottom-right-radius: 0;
+`;
+
+const DismissButton = styled(Button)`
+  border-top-left-radius: 0;
+  border-bottom-left-radius: 0;
+`;
```

Comment:
@billyvg We don't have a `ButtonGroup` component that does this type of thing yet right?

Reply:
Correct, it was a PITA last time I tried to replace our bootstrap button groups :(